packages/angular/src/date-time-picker/date-time-picker.component.ts

```diff
@@ -254,23 +316,29 @@ export class DateTimePickerComponent implements OnChanges, OnInit {
     const range = this.dateTimeRanges.find((range) => range.key === rangeOrType);
     const [start, end] = range.getRange();
     // TODO: provide a way to customize this for g11n
-    const formatString = 'yyyy-M-d HH:mm';
+    const formatString = this.dateFormat + ' ' + this.timeFormat;
     let endFormatted = format(end, formatString);
     if (isThisMinute(end)) {
-      endFormatted = 'Now';
+      endFormatted = this.batchText.NOW;
     }
-    return `${format(start, formatString)} to ${endFormatted}`;
+    return `${format(start, formatString)}${this.i18n.get().rangeSeparator}${endFormatted}`;
   }
 
   formatCustomRange() {
     // TODO: provide a way to customize this for g11n
-    const formatString = 'yyyy-M-d HH:mm';
+    const formatString = this.dateFormat + ' ' + this.timeFormat;
```

Comment:
Same template string comment here:

```suggestion
    const formatString = `${this.dateFormat} ${this.timeFormat}`;
```